src/NosCore.GameObject/Character.cs

```diff
@@ -268,7 +268,11 @@ public void SetReputation(long reput)
         {
             Reput = reput;
             SendPacket(GenerateFd());
-            Session.SendPacket(new MsgPacket { Type = MessageType.Whisper, Message = Language.Instance.GetMessageFromKey(LanguageKey.REPUTATION_CHANGED, Session.Account.Language) });
+            SendPacket(new MsgPacket
+            {
+                Type = MessageType.Whisper,
```

Comment:
Still not sure setReputation on the official send a MSG i'm quite sure it's a SayPacket